credentials/src/main/kotlin/SSI.kt

```diff
@@ -228,6 +233,220 @@ public object VerifiableCredential {
     return JwtVerifiableCredential.fromCompactSerialization(vcJWT)
       .verify_Ed25519_EdDSA(publicKeyJWK.toOctetKeyPair())
   }
+
+  /**
+   * Converts a verifiable credential JWT into a [VerifiableCredentialType] object.
+   *
+   * @param vcJwt The verifiable credential in JWT format to be converted.
+   * @return A [VerifiableCredentialType] object representing the provided verifiable credential JWT.
+   * @throws Exception Throws an exception if the conversion from JWT to [VerifiableCredentialType] fails or any other unexpected error occurs during the conversion process.
+   */
+  @Throws(Exception::class)
+  public fun fromJwt(vcJwt: VcJwt): VerifiableCredentialType {
+    return FromJwtConverter.fromJwtVerifiableCredential(JwtVerifiableCredential.fromCompactSerialization(vcJwt))
+  }
+
+  /**
+   * Generates a `StatusList2021Credential` in JWT format for a list of issued credentials.
+   *
+   * The function creates a status list credential with a specific purpose, e.g., for revocation.
+   * The credential indicates the status (like revocation status) of other credentials.
+   *
+   * @param signOptions The options required for signing the generated status list credential.
+   * @param statusListCredentialId The identifier for the status list credential.
+   * @param issuer The DID or URI of the entity issuing the status list credential.
+   * @param purpose The purpose of the status list (e.g., "revocation").
+   * @param issuedCredentialJwts A list of JWTs representing the credentials whose status needs to be represented in the status list.
+   * @return A [VcJwt] representing the `StatusList2021Credential` in JWT format.
+   * @throws Exception Throws an exception if the generation of the `StatusList2021Credential` fails or any other unexpected error occurs during the generation process.
+   */
+  @Throws(Exception::class)
+  public fun generateStatusList2021Credential(
+    signOptions: SignOptions,
+    statusListCredentialId: String,
+    issuer: String,
+    purpose: String,
+    issuedCredentialJwts: List<VcJwt>
```

Comment:
Clients that use this SDK may or may not be storing `VcJwt` objects in their DBs. As such, let's use an approach that caters to both. The name BitStringStatusListEntry comes from the [latest draft of the spec](https://w3c.github.io/vc-status-list-2021/#bitstringstatuslistentry). Crucially, this shouldn't be a secured thing, since the SDK is not performing any verification on the data structure (i.e. it should not be a JWT). 
```suggestion
    issuedCredentialStatusListEntries: Iterable<BitStringStatusListEntry >
```